### DISCUSSION TITLE
I'm a Uopeople student, how do I apply for Student pack without ID and location mismatch

### DISCUSSION THREAD

**Discussion by okomaogboi** [community/community#133757] [GitHub Education]
### Hi everyone,

Hi, I'm a student of university of the people and has applied for the student developer pack severally without success.   The problem stems mostly from the fact that we're not issued IDs and I'm in a location that doesn't match the school location.   I've tried to contact support but no help was rendered

**Comment by SohanKafle** [comment]
Hi [okomaogboi](https://github.com/okomaogboi),

If you're a University of the People student without an ID and facing location mismatch issues, try these steps:

**1. Use Alternative Proof:** Submit other forms of verification like an acceptance letter, current class schedule, or a tuition receipt.
**2. Contact School:** Ask your school for an official document verifying your student status.
**3. Explain Situation:** Clearly explain your situation in the application comments section.

If issues persist, contact [GitHub Education Support](https://support.github.com/contact/education) with detailed information about your situation.

**Reply by okomaogboi** [reply]
Thanks @SohanKafle for your response.   I've tried the 3 options.  I've tried to submit my proof of enrolment given by my school but they kept rejecting it.  I contacted the school and they told me proof of enrolment serves as ID then I contacted support still no help.  

Is there a different means I can provide my Identification and proof other than the student education pack application that seemed to be automated and automatically rejecting my application for specific reasons that might have to do with location mismatch or not getting the specific documents expected?

Please assist me with manual approval as I've tried the three options you suggested. 

Thanks

**Reply by SohanKafle** [reply]
If you've exhausted the typical options for verifying your student status and are still facing issues, here are a few additional steps you can take:

**1. Detailed Support Request:** Write a detailed email to GitHub Education Support. Include all the documents you have (proof of enrollment, acceptance letter, tuition receipt) and explain the situation thoroughly. Mention that you've tried the standard application process multiple times without success.

**2. Contact Higher Authorities:** If regular support is unhelpful, try to reach out to higher authorities within GitHub Education. Look for any escalation paths in their support system.

**3. Social Media Outreach:** Sometimes, reaching out via GitHub's official social media channels (Twitter, LinkedIn) can get attention. Explain your situation briefly and request assistance.

**4. School's IT Department:** Contact your school’s IT or administration department and explain the issue. They might have dealt with similar cases and could provide a solution or directly communicate with GitHub on your behalf.

**5. Peer Assistance:** Check if other students from your university have successfully applied and ask for their guidance. They might have specific tips or contacts.

**6. Alternative Verification:** If GitHub Education still doesn’t work out, explore other platforms offering student benefits. Some alternatives might have more flexible verification processes.

<h2> Here's a template for a detailed support request to GitHub Education:</h2>

Subject: Manual Approval Request for GitHub Student Developer Pack

Dear GitHub Education Support Team,

My name is [Your Name], and I am a student at the University of the People. I am writing to request manual approval for the GitHub Student Developer Pack as I have encountered repeated issues with the automated application process.

I have attempted to verify my student status using the following documents:

1. Proof of Enrollment provided by my university.
2. Acceptance Letter from the University of the People.
3. Current class schedule and a tuition receipt.

Unfortunately, my applications have been rejected multiple times due to reasons that seem to be related to location mismatch or document specifications. I have contacted my university, and they have confirmed that the Proof of Enrollment should serve as valid identification.

I have attached all relevant documents to this email for your review. I kindly request your assistance in manually reviewing my application and approving my access to the GitHub Student Developer Pack.

Thank you for your understanding and support.

Best regards,
[Your Name]
[Your GitHub Username]
[University of the People]
[Your Contact Information]

Attach all the relevant documents to your email. This approach might help in getting manual verification and approval.

**Comment by samus-aran** [comment]
Hi @okomaogboi 

Can you please provide your latest rejection reasons. These rejection reasons are there to provide you guidance.

I can identify that it seems that you haven't completed your Billing information fully and/or your image isn't readable. 

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

> We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.


1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
2. If you can, print off your evidence and take a photo of this copy.
3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
5. Alternatively you may use another mobile device or computer.

Please try again with a photo that is more legible and provide your rejection reasons if you are still unsuccessful 👍

**Comment by okomaogboi** [comment]
Thanks for your response.  Please, I've exhausted all the suggestions and none worked for me.  How about sending my proof of admission here for you to forward to the appropriate department?  I know this is generally a straight forward process but the fact that my school doesn't issue IDs makes it all difficult.
________________________________
